src/components/Editor/Breakpoint.js

```diff
@@ -41,7 +42,7 @@ const Breakpoint = React.createClass({
     }
   },
 
-  shouldComponentUpdate(nextProps) {
+  shouldComponentUpdate(nextProps: any) {
```

Comment:
Good question!  Essentially `nextProps` is the `propTypes` attribute of the class but I don't really want to write it out twice.  Here are the patterns I've seen in the wild when researching this.

Use the [flow-react-proptypes](https://github.com/brigand/babel-plugin-flow-react-proptypes) plugin.  This means you create a flow type for your prop types and don't define the `propTypes` in your react class; like this:

```js
type PropTypes = {
  breakpoint: Breakpoint,
  editor: any
};
```

And then babel would compile that type into a PropType for react.  This pattern doesn't settle well with me right now.  I wonder about our plans to move to es6 classes and how well this plugin works.  

Then of course other people just define both:

```js
type PropTypes = {
  breakpoint: Breakpoint,
  editor: any
};
const Breakpoint = React.createClass({
  propTypes: {
    breakpoint: PropTypes.object.isRequired,
    editor: PropTypes.object.isRequired
},
 shouldComponentUpdate(nextProps: PropTypes) {
 }
});
```

Which is a bit of overhead but would actually type the `propTypes` correctly.  Other than the overhead I don't think there is anything wrong with this pattern.  

My ideal pattern would be to type the `propTypes` inline.  So it would look something like this:

```js
const Breakpoint = React.createClass({
  propTypes: {
    breakpoint: PropTypes.object.isRequired: Breakpoint,
    editor: PropTypes.object.isRequired: any
},
 shouldComponentUpdate(nextProps: propTypes) {
 }
});
```

But so far in other components we haven't cracked this egg.